csaf-rs/src/csaf/csaf_traits.rs

```diff
@@ -75,7 +75,7 @@ pub trait DistributionTrait {
     fn get_tlp_20(&self) -> Option<&Self::TlpType>;
```

Comment:
Just saw this now, why do the `_20` functions not return a `Result` but an `Option`? Can be handled in another PR, just wondering if there is a particular reasoning.

Reply:
I don't have an opinion on that (yet). I just wanted to point of that there are value changes from TLP v1 to TLP v2...

So happy to leave it as is for now.